otb-luci/files/usr/lib/lua/luci/view/otb_overview.htm

```diff
@@ -72,6 +94,219 @@ <h2><%:Overview%></h2>
         return ret + seconds + ' seconds';
     }
 
+    /* start of otb-get-data section */
+    XHR.poll(15, '<%=url("admin/overthebox/data")%>', null, function(x,p) {
+        _otb_data = p;
+    });
+    function otb_get_data(...args) {
```

Comment:
The `...` might not work with older browser since it's a new specification

Reply:
Tested on android chrome and we are doing HTML anyway

Reply:
replaced by a real array